vehicle_profiles/toyota/rav4.json

```diff
@@ -1,20 +1,33 @@
 {
 	"car_model": "Toyota: Rav4",
-	"init": "ATSP6;ATST96;",
+	"init": "ATSP6;",
 	"pids": [
 		{
 			"pid": "015B2",
 			"parameters": [
 				{
 					"name": "SoC",
+					"pid_init": "ATST96;",
 					"expression": "(B3*20)/51",
 					"unit": "%",
 					"class": "battery"
 				}
 			]
 		},
 		{
-			"pid":"01311",
+			"pid": "01A61",
+			"parameters": [
+				{
+					"name": "Odometer",
+					"pid_init": "ATST96;",
+					"expression": "(([B3:B6])/10)/1.609",
+					"unit": "mi",
```

Comment:
IMO were best to leave it as Kilometers, since its much more internationally standard, and various integrations can convert to miles if needed